collector/src/main.rs

```diff
@@ -1252,6 +1252,9 @@ fn download_from_git(target: &Path, url: &str) -> anyhow::Result<()> {
         .status()
         .expect("Git clone failed");
     generate_lockfile(tmpdir.path());
+
+    std::fs::remove_dir_all(tmpdir.path().join(".git")).expect("Could not delete .git directory");
```

Comment:
I think it would be better to just log an error with `eprintln!` if deleting the directory fails instead of crashing. After all, including `.git` isn't incorrect, it's just a bit wasteful. What do you think?

Reply:
I stumbled upon the `.git` directory when I was adding a new benchmark. If the `.git` directory is left there, it will be problematic to work with the directory using `git`, for example if I wanted to add the downloaded benchmark to the `rustc-perf` suite. But in most cases crates downloaded from `git` probably won't be added to the repository anyway, so it's not a big deal.